ocaml/xapi/xapi_vdi.ml

```diff
@@ -25,6 +25,30 @@ open Printf
 (**************************************************************************************)
 (* current/allowed operations checking                                                *)
 
+let check_sm_feature_error op sm_features sr =
```

Comment:
Hmm, then it is subjective, I personally prefer good & semantic code structure to smaller functions, cause nothing fits into my screen :laughing: